version.php

```diff
@@ -24,8 +24,8 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2017081700;    // The current plugin version (Date: YYYYMMDDXX).
-$plugin->release   = 2017081700;    // Match release exactly to version.
+$plugin->version   = 2017101700;    // The current plugin version (Date: YYYYMMDDXX).
```

Comment:
@brendanheywood I'm not sure what your version bumping strategy is, but I just realized that this version is higher than the version in the 3.3 branch that updates the configs to the 3.3 style.

Reply:
Ah crap yeah, I think there is a larger problem with the two stables using todays date instead of the fork date + increment. ala https://moodle.org/mod/forum/discuss.php?d=275635#p1184986

I think we'll just have to pick two arbitrary dates for each stable instead of the fork date and then increment from there.